src/lib/components/table/proposals/ResolvedHeight.tsx

```diff
@@ -21,6 +23,7 @@ export const ResolvedHeight = ({
       type="block_height"
       value={resolvedHeight.toString()}
       showCopyOnHover
+      ampCopierSection={amptrackSection}
```

Comment:
just hardcode proposal list if its only used in that page

Reply:
also use in account details page